components/visuals/TravelsByDepartmentPrismMap.tsx

```diff
@@ -0,0 +1,112 @@
+import { scaleOrdinal, scaleLinear } from "d3";
+import { GeoProjection } from "d3-geo";
+import { GeoJsonProperties } from "geojson";
+import { FC, useState } from "react";
+import { ExtrudeGeometryOptions } from "three";
+import type { Topology } from "topojson-specification";
+import PrismMap from "../PrismMap";
+import clsx from "clsx";
+import {
+  AccumulativeShadows,
+  Environment,
+  OrbitControls,
+  RandomizedLight,
+} from "@react-three/drei";
+import { Canvas } from "@react-three/fiber";
+import { BtorsGroupedByCountryByDepartment } from "../../lib/data/queries/btors/getBtorsGroupedByCountryByDepartment";
+
+type Props = {
+  btorsByCountryByDepartment: BtorsGroupedByCountryByDepartment;
+  topology: Topology;
+  topologyObject: string;
+  projection: GeoProjection;
+  width: number;
+  length: number;
+  extrudeGeometryOptions?: ExtrudeGeometryOptions;
+};
+
+const TravelsByDepartmentPrismMap: FC<Props> = ({
+  btorsByCountryByDepartment,
+  topology,
+  topologyObject,
+  projection,
+  width,
+  length,
+  extrudeGeometryOptions = {},
+}) => {
+  const ITC = ["EOS", "AES", "GIP", "NRS", "PGM", "WRS", "VAR"];
```

Comment:
You cant get this from the database 🙂 We already have a query for that (see `getDepartments`)!